nwb_conversion_tools/datainterfaces/roiextractordatainterface.py

```diff
@@ -0,0 +1,36 @@
+from abc import ABC
+
+from roiextractors import CnmfeSegmentationExtractor, ExtractSegmentationExtractor, \
+    CaimanSegmentationExtractor, Suite2pSegmentationExtractor, SimaSegmentationExtractor
+
+from ..basesegmentationextractorinterface import BaseSegmentationExtractorInterface
+
+
+class CnmfeSegmentationInterface(BaseSegmentationExtractorInterface, ABC):
```

Comment:
why are these all ABC?

Reply:
yeah my mistake, it has no use here.  
In that case should `BaseSegmentationExtractorInterface`/`BaseRecordingExtractorInterface`/`BaseSortingExtractorInterface`  continue to inherit ABC? Is it of any use in the absence of any `@abstractmethod`?
Pythonically, I think, `BaseDataInterface` ([here](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/nwb_conversion_tools/basedatainterface.py#L9)) which has `@abstractmethod` should inherit from ABC (it doesn't currently) as this would prevent it from being instantiated.

Reply:
the base classes should remain ABC I think

Reply:
fixed https://github.com/catalystneuro/nwb-conversion-tools/pull/108/commits/c11bc550cfa7bfc2e6bc7db2f33f8d98472556db